src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__month month month--length-31 month--start-day-sun">
```

Comment:
```suggestion
    <div class="page__month month month--31 month--sun">
```
Можно короткие классы сделать